README.md

```diff
@@ -1,65 +1,151 @@
 
-If you have any questions, please reach out to us on [Mozilla Developer Network](https://developer.mozilla.org).
+Create a configuration file by copying the [`config/config-example.json`](conf/config-example.json) to `config/config.json`.
+Put in your database credentials into `config/config.json`:
 
-<!--
+```json
+{
+  "database": {
+    "database": "observatory",
+    "user": "postgres"
+  }
+}
 
-TODO:
+```
 
-Details (with links) to meetings, mailing lists, Slack, and any other communication channels]
+To initialize the database with the proper tables, use this command to migrate. This is a one-time action, but future code changes
+might need further database changes, so run this migration every time the code is updated from the repository.
 
-- User Mailing List:
-- Developer Mailing List:
-- Slack Channel:
-- Public Meeting Schedule and Links:
-- Social Media:
+```sh
+npm run migrate
+```
 
--->
+Finally, start the server by running
 
-## License
+```sh
+npm start
+```
 
-This project is licensed under the [Mozilla Public License 2.0](LICENSE).
+The server is listening on your local interface on port `8080`. You can check the root path by opening <http://localhost:8080/> in your browser or `curl` the URL. The server should respond with `Welcome to the MDN Observatory!`.
+
+## JSON API
+
+**Note:** We provide these endpoints on our public deployment of HTTP Observatory at <https://observatory-api.mdn.mozilla.net/>
+
+### POST `/api/v2/scan`
+
+For integration in CI pipelines or similar applications, a JSON API endpoint is provided. The request rate is limited to one scan per host per `api.cooldown` (default: One minute) seconds. If exceeded, a cached result will be returned.
+
+#### Query parameters:
+
+* `host` hostname (required)
+
+#### Examples:
+
+* `POST /api/v2/scan?host=mdn.dev`
+* `POST /api/v2/scan?host=google.com`
 
-<!--
+#### Result
 
-We generally use the Mozilla Public License 2.0 and CCSA 2.5 licenses for our projects, see: https://github.com/mdn/content/blob/main/LICENSE.md.
+On success, a JSON object is returned, structured like this example response:
 
-This template is on based on the [CNCF project template](https://github.com/cncf/project-template) distributed under an [Apache license 2.0](https://github.com/cncf/project-template/blob/main/LICENSE).
+```json
+{
+  "id": 77666718,
+  "details_url": "https://developer.mozilla.org/en-US/observatory/analyze?host=mdn.dev",
+  "algorithm_version": 4,
+  "scanned_at": "2024-08-12T08:20:18.926Z",
+  "error": null,
+  "grade": "A+",
+  "score": 105,
+  "status_code": 200,
+  "tests_failed": 0,
+  "tests_passed": 10,
+  "tests_quantity": 10
+}
+```
 
--->
+**Note:** For a full set of details about the host, use the provided link in the `details_url` field.
+
+If an error occurred, an object like this is returned:
+
+```json
+{
+  "statusCode": 422,
```

Comment:
Having a `status_code` field in the successful response (which comes from the host being scanned), and a `statusCode` response in the failing response (which comes from us), seems a bit confusing - do we need this `statusCode` field?

Reply:
Agreed, I'll take the status out for the error. It is part of the HTTP response anyway.